plugins/frontend/backstage-plugin-cloudsmith/README.md

```diff
@@ -57,6 +57,7 @@ proxy:
     target: 'https://api.cloudsmith.io/v1'
     headers:
       X-Api-Key: ${CLOUDSMITH_API_KEY}
+      User-Agent: 'Roadie-Backstage'
```

Comment:
This should be just Backstage. It can be used by backstage deployments other than roadie.